portal/views/assessment_engine.py

```diff
@@ -1575,6 +1575,10 @@ def assessment_add(patient_id):
     db.session.commit()
     questionnaire_response.assign_qb_relationship(
         acting_user_id=current_user().id)
+
+    from ..tasks import extract_observations_task
```

Comment:
shouldn't we limit this expensive call to QNRs that include the `sdc-questionnaire-observationExtract` extension?

Reply:
Thanks, as discussed added a hack and a TODO so we don't forget :crossed_fingers: